client/reader/discover/post-attribution.jsx

```diff
@@ -61,7 +61,13 @@ class DiscoverPostAttribution extends React.Component {
 					? <img className="gravatar" src={ encodeURI( attribution.avatar_url ) } alt="Avatar" width="20" height="20" />
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?